.github/workflows/windows.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       - name: Tests
         run: |
           New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
+          git config --system core.longpaths true
```

Comment:
you should use https://github.com/xmake-io/xmake-repo/blob/08176e673a9ea51a59240c561f9eb46fc01806cc/packages/l/libtorch/xmake.lua#L48

do not modify ci.

Reply:
Thanks for pointing it out!